tests/golden/apiserver/appcat/appcat/10_appcat_metering_recording_rule.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: appcat-metering
+  name: appcat-metering
+  namespace: syn-appcat
+spec:
+  groups:
+    - name: appcat-metering-rules
+      rules:
+        - expr: |
+            sum by (label_appcat_vshn_io_claim_namespace,label_appcat_vshn_io_sla,label_appuio_io_billing_name,label_appcat_vshn_io_claim_name,sales_order,label_appuio_io_billing_name) (
+              # Sum values over one hour and get mean
+              sum_over_time(
+                # Default appcat.vshn.io/sla to besteffort if it is not set
+                label_replace(
+                    # Fetch all namespaces with label label_appuio_io_billing_name=~"appcat-.+"
+                    (
+                      # We hardcode the tenant ID here, so that we have the label to do the join.
+                      label_replace(
+                          kube_namespace_labels{ label_appuio_io_billing_name=~"appcat-.+"} *
+                          on (namespace) group_right(label_appcat_vshn_io_claim_namespace,label_appcat_vshn_io_sla, label_appuio_io_billing_name,label_appcat_vshn_io_claim_name,sales_order,label_appuio_io_billing_name)
+                          kube_pod_info{created_by_kind!="Job"},
+                        "sales_order",
+                        "10431",
+                        "",
+                        ""
+                      )
+                    ),
+                  "label_appcat_vshn_io_sla",
+                  "besteffort",
+                  "label_appcat_vshn_io_sla",
+                  "^$"
+                )[60m:1m]
+              )/60
+            )
+          record: appcat:metering
```

Comment:
have You tested this query against prod prometheuses?

Reply:
yes